mobingi/sesha3/sesha3.go

```diff
@@ -0,0 +1,34 @@
+package sesha3
+
+import (
+	"github.com/moul/gotty-client"
+	"github.com/pkg/errors"
+)
+
+type SeshaClientInput struct {
+	URL string
+}
+
+type Creds struct {
+	client *gottyclient.Client
+}
+
+func New(in *SeshaClientInput) (*Creds, error) {
```

Comment:
@biosugar0 Can you change this from `New` to `NewClient`? There will be another `New...` in the future under this package.